pkg/idtools/utils_unix.go

```diff
@@ -26,7 +25,7 @@ func resolveBinary(binname string) (string, error) {
 	return "", fmt.Errorf("Binary %q does not resolve to a binary of that name in $PATH (%q)", binname, resolvedPath)
 }
 
-func execCmd(cmd, args string) ([]byte, error) {
-	execCmd := exec.Command(cmd, strings.Split(args, " ")...)
+func execCmd(cmd string, args []string) ([]byte, error) {
```

Comment:
Actually, perhaps just change this to match `exec.Command`

```suggestion
func execCmd(cmd string, arg ...string) ([]byte, error) {
```